doc/source/Windows.rst

```diff
@@ -0,0 +1,106 @@
+Windows Tutorial 
+================
+
+This guide gives you a brief introduction to how volatility3 works and some demonstration on suite of plugins available from
+
+Acquiring memory
+----------------
+
+Volatility does not provide the ability to acquire memory. In this tutorial we will see how we can use  `WinPmem <https://github.com/Velocidex/WinPmem/releases/latest>`_ for this purpose.
+
+Listing Plugins
+---------------
+
+.. code-block:: shell-session
+
+    $ python3 vol.py --help | grep windows | head -n 5
+        windows.bigpools.BigPools
+        windows.cmdline.CmdLine
+        windows.crashinfo.Crashinfo
+        windows.dlllist.DllList
+                            Lists the loaded modules in a particular windows
+
+Using plugins
+-------------
+
+The following is the syntax to run volatility tool.
+
+.. code-block:: shell-session
+
+    $ python3 vol.py -f <path to memory image> plugin_name plugin_option
+
+
+Example
+-------
+
+Example 1
+~~~~~~~~~
+
+In this example we will be using memory dump from PragyanCTF'22. The dump is available `here <https://drive.google.com/file/d/1VPIdqIDFkOi3zGET2g00s1y-OeWOUqyi/view?usp=sharing>`_.
+We will limit the discussion to memory forensics with volatility3 and not extend to other parts of the challenges. 
+
+In windows memory forensics using volatility3, most of the times we do not require creating a ISF file. 
+
+.. code-block:: shell-session
+
+    $ python3 vol.py -f MemDump.DMP windows.pslist | head -n 10
+        
+        Volatility 3 Framework 2.0.2	PDB scanning finished                                
+
+        PID	PPID	ImageFileName	Offset(V)       Threads	Handles	SessionId	Wow64	CreateTime	    ExitTime            File output
+
+        4	0	    System	        0xfa8000cbc040	85	    492	    N/A	        False	2022-02-07      16:30:12.000000 	N/A	Disabled
+        276	4	    smss.exe	    0xfa8001e04040	2	    29	    N/A	        False	2022-02-07      16:30:12.000000 	N/A	Disabled
+        352	336	    csrss.exe	    0xfa8002110b30	9	    375	    0	        False	2022-02-07      16:30:13.000000 	N/A	Disabled
+        404	336	    wininit.exe	    0xfa800219f060	3	    74	    0	        False	2022-02-07      16:30:13.000000 	N/A	Disabled
+        412	396	    csrss.exe	    0xfa80021c5b30	9	    224	    1	        False	2022-02-07      16:30:13.000000 	N/A	Disabled
+        468	396	    winlogon.exe    0xfa8002284060	5	    113	    1	        False	2022-02-07      16:30:14.000000 	N/A	Disabled
+
+``windows.pslist`` helps us list the processes running while the memory dump was taken.
+
+.. code-block:: shell-session
+
+    $ python3 vol.py -f MemDump.DMP windows.pstree | head -n 20
+        Volatility 3 Framework 2.0.2	PDB scanning finished                                
```

Comment:
Same context as above. (version) :)

Reply:
Description on why the pipe is used added in note section